test/CAS/mccas.swift

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: OS=macosx
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -c %s -g -cas-backend -cas-backend-mode=verify -cas-path %t/cas -o %t/test-verify.o
+// RUN: %llvm-dwarfdump %t/test-verify.o | %FileCheck %s --check-prefix=VERIFY-FILE
+// VERIFY-FILE: .debug_info
+
+// RUN: %target-swift-frontend -c %s -g -cas-backend -cas-backend-mode=native -cas-path %t/cas -o %t/test-native.o
+// RUN: %llvm-dwarfdump %t/test-native.o | %FileCheck %s --check-prefix=NATIVE-FILE
+// NATIVE-FILE: .debug_info
+
+// RUN: %target-swift-frontend -c %s -g -cas-backend -cas-backend-mode=casid -cas-path %t/cas -o %t/test-casid.id
+// RUN: cat %t/test-casid.id | %FileCheck %s --check-prefix=CASID-FILE
+// CASID-FILE: CASID:Jllvmcas://{{.*}}
```

Comment:
`Jllvmcas://`? I won't use that. I assume J is coming from the VBR8 encoding and it is tied to the length of the CASID, which should not be assumed to be constant (since it is configureable) and it is also unnecessary to check that.